src/app/components/Figure/Copyright/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { node } from 'prop-types';
+import styled from 'styled-components';
+import {
+  FF_NEWS_SANS_REG,
+  C_WHITE,
+  C_EBON,
+  GEL_SPACING,
+} from '../../../lib/constants/styles';
+import { T_MINION } from '../../../lib/constants/typography';
+
+// const copyrightSymbolPrefix = '&#169; ';
+const copyrightSymbolPrefix = '© ';
```

Comment:
Curious on other reviewers opinion of if we should use the encoded version of this symbol or keep as is

Reply:
How sure are we that using the encoded one will work when it is sent to the client?